sord.gemspec

```
@@ -24,8 +24,9 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'yard'
   spec.add_dependency 'sorbet-runtime'
-  spec.add_dependency 'commander', '~> 4.5'
-  spec.add_dependency 'parlour', '~> 5.0'
+  spec.add_dependency 'commander', '~> 5.0'
+  spec.add_dependency "parser"
+  spec.add_dependency 'parlour', '~> 9.0'
```

This could be updated to whatever the new version number is once published for that heredoc support.

---

This is now released as Parlour 9.1

```suggestion
  spec.add_dependency 'parlour', '~> 9.1'
```